app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
```

Comment:
can we use JSON to remove the extra require? just generally want to avoid extra stuff if we can